### DISCUSSION TITLE
Getting error messages about mp3 playback using ESP32-audioI2S library

### DISCUSSION THREAD

**Discussion by Xk-w** [schreibfaul1/ESP32-audioI2S#602] [Q&A]
When I use ESP32-audioI2S library for mp3 music playback and unplug the SD card, how should I get the message that the SD card has been unplugged in ESP32-audioI2S library?

**Comment by biologist79** [comment]
There are SD-slots available, that detect if card is inserted or not. So you need to read in that state using a GPIO or port expander and handle it accordingly.
